src/device_trezor/trezor/protocol.hpp

```diff
@@ -92,11 +92,13 @@ namespace protocol{
 // Crypto / encryption
 namespace crypto {
 namespace chacha {
+  const unsigned IV_SIZE = 12;
```

Comment:
That's going to be super confusing given the chacha iv size is 8 in chacha.h...

Reply:
I will add a comment it is defined in RFC 7539. It defines 96-bit nonce/IV